### PR TITLE
Initialize project

### PR DESCRIPTION
**Pull request by Thoth1111**
### "It's Go Time!! 🚀 
In this branch I initialized my project by doing the following in my commits: 

✅ Created the React App and removed the boilerplate files and code
✅ Added components for the App, Book, Form & Navbar
✅ Setup my React router and added the Books and Categories routes
✅ Modified the Navbar component to include Links to the two routes.
✅ Deployed the development branch on Render which will keep track and redeploy based on changes to that branch
✅ Updated the README.md file
✅ Added Linter check files and fixed linter errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,150 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
 
-### Deployment
+<!-- LICENSE -->
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+## 📝 License <a name="license"></a>
 
-### `npm run build` fails to minify
+This project is [MIT](https://github.com/Thoth1111/my-book-store/blob/develop/LICENSE) licensed.
```
Comment (elisha2kyakpo1) [inline README.md]: - [ ] When I clicked the license link, I got a 404 error message. Kindly fix the broken link to make the readme file look more professional.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @Thoth1111, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description look good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Thoth1111) [reply README.md]: Hi Elisha. 
Okay, thanks. I've fixed the issue.

**Comment by arslanbisharat** [comment]
Hi @Thoth1111,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**